facade/interface.go

```diff
@@ -129,7 +129,7 @@ type FacadeInterface interface {
 
 	GetAllServiceDetails(ctx datastore.Context) ([]service.ServiceDetails, error)
 
-	GetServiceDetails(ctx datastore.Context, serviceID string) (*service.ServiceDetails, error)
+	GetServiceDetails(ctx datastore.Context, serviceID string, ancestors bool) (*service.ServiceDetails, error)
```

Comment:
Rather than have the bool parameter, have a GetServiceDetailsAncestors method that takes a service and returns the ancestors.